sources/packages/backend/apps/api/src/route-controllers/supporting-user/supporting-user.supporting-users.controller.ts

```diff
@@ -111,13 +111,16 @@ export class SupportingUserSupportingUsersController extends BaseController {
         ),
       );
     }
+    
```

Comment:
Can you please make sure that required extensions to VS code are installed, settings.json is updated(any dev can share that) and also the EOL git command is run?

Wiki can help here.
https://github.com/bcgov/SIMS/wiki/Development---Standards,-Decisions,-and-Troubleshooting#development-environment-setup

Reply:
VS code extensions and settings are properly configured. Please have a look.